### ISSUE TITLE
Specify Environment Variables for aws batch

### ISSUE THREAD

**Issue by egafni** [Mizzou-CBMI/COSMOS2#143]
add parameter and attribute:
```
Cosmos.__init__(default_environment_variables)
```
add parameter to:
```
def Workflow.add_task(environment_variables)
  if is None:
    default_environment_variables = environment_variables 
  task.environment_variables = environment_variables
```
create column
```
Task.enviroment_variables (use the mutable json dict column type used for the params column)
```

Add environment_variables display to task.html

Then you can pass environment_variables to submit_job in drm_aws_batch.  

Be sure to get it working with the local drm too, and write a test (just have a one Task workflow which checks os.environ()).  for other drms like slurm/sge/lsf, raise a NotImplemented exception if environment_variables is None.

**Comment by danielgafni** [comment]
> for other drms like slurm/sge/lsf, raise a NotImplemented exception if environment_variables is None.

You mean not None?

**Comment by danielgafni** [comment]
Looks like `drm_awsbatch` already is set up to work with additional environment variables, but they were never used (always None). Is this correct? Than the only thing I have to do here is pass them from `task.environment_variables`

```python
def submit_script_as_aws_batch_job(
    local_script_path,
    s3_prefix_for_command_script_temp_files,
    job_name,
    job_def_arn,
    job_queue,
    instance_type=None,
    memory=1024,
    vpu_req=1,
    gpu_req=None,
    environment=None,
    tags=None,
):
    """
    :param local_script_path: the local path to a script to run in awsbatch.
    :param s3_prefix_for_command_script_temp_files: the s3 bucket to use for storing the local script to to run.  Caller
      is responsible for cleaning it up.
    :param job_name: name of the job_dict.
    :param container_image: docker image.
    :param memory: amount of memory to reserve.
    :param vpu_req: amount of vcpus to reserve.
    :param environment: {env_name -> env_val} environment variables to set  <--- here!!!
    :return: obId, job_definition_arn, s3_command_script_uri.
    """
```

**Comment by egafni** [comment]
> > for other drms like slurm/sge/lsf, raise a NotImplemented exception if environment_variables is None.
> 
> You mean not None?

I mean None.  You should raise a NotImplemented in since a user expects environment variables to work with those DRMs, but its not implemented

**Comment by egafni** [comment]
Fixed in #145
